src/index.html

```diff
@@ -1,13 +1,50 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      >
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700& display=swap"
+      rel="stylesheet"
+    >
+  </head>
+
+  <body>
+    <div class="card card--size" data-qa="card">
+      <img
+        class="card__logo"
+        src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+      >
+      <section>
+        <h1>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+        <p class="product-code">Product code: 195434</p>
+      </section>
+
+      <section class="section-stars">
+        <div class="section-stars__section section-stars__section--star-4">
+          <div class="section-stars__star"></div>
+          <div class="section-stars__star"></div>
+          <div class="section-stars__star"></div>
+          <div class="section-stars__star"></div>
+          <div class="section-stars__star"></div>
+        </div>
+        <p class="section-stars__reviews">Reviews: 5</p>
+      </section>
+
+      <section class="section-price">
+        <p class="section-price__info">Price:</p>
+        <p class="section-price__price">$2,199</p>
+      </section>
```

Comment:
same here with class 
```suggestion
      <section class="card__price-block">
        <p class="card__info">Price:</p>
        <p class="card__price">$2,199</p>
      </section>
```